openapi.yaml

```diff
@@ -525,6 +569,39 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/TransmissionType"
+    SellerResponse:
+      description: Represents a set of fields for Seller object.
+      properties:
+        address:
+          description: Seller's address.
+          type: string
+          example: "Bahnhofstrasse 123b"
+        city:
+          description: Seller's city.
+          type: string
+          example: "Zurich"
+        id:
+          description: Seller's ID.
+          type: integer
+          format: int64
+          example: 1923
+        name:
+          description: Seller's name.
+          type: string
+          example: "John Seller LLC"
+        phone:
+          description: Seller's phone number.
+          type: string
+          example: "+41761234567"
+        phones:
+          description: All seller's phone numbers.
+          type: array
+          items:
+            $ref: "#/components/schemas/Phone"
```

Comment:
your suggestion from slack to go with `primary phone` or similar fits better indeed

we could go with `All seller`s phones` for the second one maybe? 
because it's an array of Phone objects, that have a number as property - would be a bit misleading to use `All seller's phone numbers.` in my oppinion 

wdyt?